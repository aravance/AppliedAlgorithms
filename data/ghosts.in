2
0.0 0.0
3.0 4.5
0.0 3.1
2.5 0.0
3
0.0 4.0
3.0 0.0
-3.0 0.0
0.0 -4.0
1.0 4.0
2.5 -3.5
8
0.0 3.1
2.5 0.0
0.0 2.0
3.0 6.0
1.0 5.0
1.0 3.0
1.8 2.2
2.0 3.0
0.0 0.0
3.0 4.5
4.0 2.0
3.0 1.0
1.0 4.0
3.0 3.0
2.0 6.0
0.0 4.0
0
