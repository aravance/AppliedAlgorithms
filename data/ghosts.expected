Data Set #1
(0.0,0.0)->(2.5,0.0)
(3.0,4.5)->(0.0,3.1)
Data Set #2
(-3.0,0.0)->(0.0,-4.0)
(0.0,4.0)->(1.0,4.0)
(3.0,0.0)->(2.5,-3.5)
Data Set #3
(0.0,2.0)->(1.0,4.0)
(0.0,3.1)->(2.0,6.0)
(1.0,3.0)->(0.0,0.0)
(1.0,5.0)->(0.0,4.0)
(1.8,2.2)->(3.0,1.0)
(2.0,3.0)->(3.0,3.0)
(2.5,0.0)->(4.0,2.0)
(3.0,6.0)->(3.0,4.5)

